extension/src/background/messageListener/freighterApiMessageListener.ts

```diff
@@ -179,7 +186,7 @@ export const freighterApiMessageListener = (
     const server = stellarSdkServer(networkUrl);
 
     try {
-      await server.checkMemoRequired(transaction);
+      await server.checkMemoRequired(transaction as any);
```

Comment:
curious that we have to cast these as any after the types come through. I don't have a suggestion yet but seems like this should be an `xdr.Transaction` maybe?